src/luapatt.py

```diff
@@ -19,8 +19,9 @@
 # IN THE SOFTWARE.
 
 from array import array
+import unicodedata
 
-__version__ = '0.9.0b5'
+__version__ = '0.9.0b6'
```

Comment:
The version change should be reverted. I'll handle the version update myself as part of the release process, and the next release might be something different (such as 1.0).

```suggestion
__version__ = '0.9.0b5'
```